app/build.gradle

```diff
@@ -0,0 +1,79 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 28
```

Comment:
Use Android 10
```suggestion
    compileSdkVersion 29
```

Reply:
If we target to Android 10, `Environment.getExternalStorageDirectory` is deprecated. It is the method used to find the path for `Fate-Grand-Automata` folder where support images are stored.
On Android 10, we can use `requestLegacyStorage` manifest option for now but won't work with Android 11.